certification/internal/engine/engine.go

```diff
@@ -435,48 +435,47 @@ func getBgName(srcrpm string) string {
 }
 
 func writeRPMManifest(ctx context.Context, containerFSPath string) error {
-	pkgList, err := rpm.GetPackageList(ctx, containerFSPath)
-	if err != nil {
-		return err
-	}
+	rpms := make([]pyxis.RPM, 0)
 
-	// covert rpm struct to pxyis struct
-	rpms := make([]pyxis.RPM, 0, len(pkgList))
-	for _, packageInfo := range pkgList {
-
-		var bgName, endChop, srpmNevra, pgpKeyId string
-
-		// accounting for the fact that not all packages have a source rpm
-		if len(packageInfo.SourceRpm) > 0 {
-			bgName = getBgName(packageInfo.SourceRpm)
-			endChop = strings.TrimPrefix(strings.TrimSuffix(regexp.MustCompile("(-[0-9].*)").FindString(packageInfo.SourceRpm), ".rpm"), "-")
+	pkgList, err := rpm.GetPackageList(ctx, containerFSPath)
```

Comment:
This all seems like a lot of churn. It seems that perhaps handling the error differently at the call site might make for less change? It might mean more places to change, but less overall code.

Reply:
reverted this, and just removed the `return err` and log the error instead.